### DISCUSSION TITLE
Proprietary Data only 16 chars long

### DISCUSSION THREAD

**Discussion by davidcallen** [open-license-manager/licensecc#168] [General]
Proprietary Data only 16 chars long. Seems quite short.  I was hoping to be able to store information in here like "max_number_of_users", "max_cpus", "issued_to_company_name". 

So with the 16 char restriction, I may still be able to use licensecc but only storing really essential info in ProprietaryData, and then in a very short format (i.e. not json, xml etc...).

Feels like I may have wasted quite a few hours reading the docs, compiling licensecc and tgrying to integrate it. Frustrating.

Maybe I am missing something here .. is there another place for storing a reasonable size of info and in a pleasant format ?

**Comment by gcontini** [comment]
I will expand to 64 chars. Issue #198. I want to keep the license size to a minimum, since it is designed to be passed in an environment variable, and a license file can contain many features... on windows the limit is about 1024 chars.

**Comment by gcontini** [comment]
By the way, there is a file in your projects folder `<PROJECT_NAME>/include/licensecc/<PROJECT_NAME>/build_properties.h`

This file contains all the specific settings for your project and it is safe for you to configure (it won't be overwritten, it is ignored by GIT).  This file contains the definition of `LCC_API_PROPRIETARY_DATA_SIZE` that is what you want to change.
